Build Failure: CMake version compatibility

#### Description
The build process for the `scangearmp2-sane-git` package fails due to a CMake version compatibility issue. The error occurs during the `build()` step, as the CMakeLists.txt file specifies a minimum required CMake version that is incompatible with the installed version of CMake.

#### Steps to Reproduce
1. Run the following command to install the package:
   ```bash
   paru -S scangearmp2-sane-git
   ```
2. Proceed with the installation and build process.
3. Observe the error during the `build()` step.

#### Error Log
```
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.

-- Configuring incomplete, errors occurred!
==> ERROR: A failure occurred in build().
    Aborting...
```

#### Environment
- **Package Version**: `4.60.2+2.r204.20230920.78ec743-1`
- **CMake Version**: `4.0.0-1`
- **OS**: Arch Linux
- **Build Tool**: `paru`

#### Possible Cause
The CMakeLists.txt file specifies a `cmake_minimum_required` version that is incompatible with the installed version of CMake. The build process fails because compatibility with older versions of CMake has been removed.

#### Suggested Fix
1. Update the CMakeLists.txt file to specify a compatible minimum CMake version.
https://github.com/ThierryHFR/scangearmp2/blob/3fa94c145968f4219c4ff81d3406bdf644604ae5/CMakeLists.txt#L1
   - Example: Update the `cmake_minimum_required` version to `3.5` or higher.
2. Alternatively, add the following flag to the build process to bypass the compatibility check:
   ```bash
   -DCMAKE_POLICY_VERSION_MINIMUM=3.5
   ```
3. Ensure that the installed version of CMake meets the updated requirements.

#### Additional Notes
- The issue may require coordination with the package maintainer to update the PKGBUILD or source files.
- If the issue persists, consider downgrading CMake to a compatible version or updating the source code to align with the latest CMake policies.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.4)
+cmake_minimum_required(VERSION 3.5)
 
 project(scangearmp2-sane)
 

